h5py_wrapper/wrapper.py

```diff
@@ -339,7 +339,9 @@ def _array_to_type(value, value_type):
                   'int': 'int',
                   'str': 'str',
                   'bool': 'bool',
-                  'Quantity': 'pq.Quantity'}
+                  'Quantity': 'pq.Quantity',
+                  'int64': 'np.int64',
+                  'float64': 'np.float64'}
```

Comment:
I am wondering what happens to `np.int32` types. Are they just detected as `int`? Or would the wrapper fail? Could you also test for that?